lambdas/main.go

```diff
@@ -314,10 +318,19 @@ func getClusterStatus(ctx context.Context, stateTable, stateTableHashKey, stateK
 	if managementLambdaName == "" {
 		return protocol.ClusterStatus{}, fmt.Errorf("MANAGEMENT_LAMBDA is not set")
 	}
+	usernameId := os.Getenv("USERNAME_ID")
+	deploymentPasswordId := os.Getenv("DEPLOYMENT_PASSWORD_ID")
+	adminPasswordId := os.Getenv("ADMIN_PASSWORD_ID")
+	creds, err := common.GetDeploymentOrAdminUsernameAndPassword(usernameId, deploymentPasswordId, adminPasswordId)
+	if err != nil {
+		return protocol.ClusterStatus{}, fmt.Errorf("getClusterStatus > GetDeploymentOrAdminUsernameAndPassword: %w", err)
+	}
 	managementRequest := management.ManagementRequest{
 		Type: "status",
 		StatusRequest: management.StatusRequest{
 			BackendPrivateIps: ips,
+			Username:          creds.Username,
+			Password:          creds.Password,
```

Comment:
Do we also want to pass credentials here? or do we want to limit this PR to just the clusterization check?